src/snippets/snippets.service.ts

```diff
@@ -22,7 +31,7 @@ export class SnippetsService {
   update(id: string, updateSnippetDto: UpdateSnippetDto) {
     return this.prismaService.snippet.update({
       where: { id },
-      data: updateSnippetDto,
+      data: { ...updateSnippetDto, updatedAt: new Date(Date.now()) }
```

Comment:
Zmiana `updatedAt` to trochę bardziej odpowiedzialnosć repozytorium a nie samego serwisu. Może warto tutaj wprowadzić jakąś warstwę dostępu do danych, tak jak np. tu https://stackoverflow.com/a/70806772, która opakuje nam prismę?